plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/server/debug/OpenShiftDebugMode.java

```diff
@@ -155,25 +161,114 @@ public OpenShiftDebugMode disableDevmode() {
 	 * @throws CoreException
 	 */
 	public OpenShiftDebugMode execute(IProgressMonitor monitor) throws CoreException {
-		IDeploymentConfig dc = getDeploymentConfig(context, monitor);
+		Connection connection = OpenShiftServerUtils.getConnection(context.getServer());
+		IResource resource = OpenShiftServerUtils.getResource(context.getServer(), monitor);
+		IDeploymentConfig dc = getDeploymentConfig(resource, connection, monitor);
```

Comment:
What if there is no DC but an RC ?

Reply:
that's part of another jira that we have.